### DISCUSSION TITLE
Notesnook 3rd audit!

### DISCUSSION THREAD

**Discussion by Devious3901** [streetwriters/notesnook#3297] [Q&A]
This is a message to the Notesnook team!
I'm happy with your app, it's great and better than the competition. You know about Standard notes, but the reason why I trust them a little more is the independent code audit!
Please hire some organization for a 3rd party audit! It would be a good sign for me and many other users to switch to notesnook completely, because the certified encryption and all other features are fully present and fully fulfill the functions of protecting users' personal data!
Add AUDIT to your RoadMap!

**Comment by cortadocodes** [comment]
Adding this here for consolidation https://github.com/streetwriters/notesnook/issues/2066
